### PR TITLE
Added one additional test to test folder

### PR DESCRIPTION
**Pull request by monito444**
Also made a few fixes.

### CODE REVIEW

File: test/project/TestNegativeCompute.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+public class TestNegativeCompute {
+	// Test the high values and negative input
+	@Test
+	public void testNegativeValue() {
+		ComputeEngine engine = new ComputeEngineImp();
+		Assertions.assertEquals("188", engine.compute(-13));
```
Comment (azip3) [inline test/project/TestNegativeCompute.java:11]: Very Good. Nit would be to just add more cases and edge cases, but negatives are already its own edge case so the small negative and larger negative seems valid.

File: test/project/TestComputeEngine.java [hunk 2]
```diff
@@ -8,6 +8,6 @@ public class TestComputeEngine {
 	@Test
 	public void testCompute() {
 		ComputeEngine engine = new ComputeEngineImp();
-		Assertions.assertEquals("1", engine.compute(1));
+		Assertions.assertEquals("0", engine.compute(1));
```
Comment (azip3) [inline test/project/TestComputeEngine.java:11]: Good. NIt would be that this is a small amount of test cases

File: test/project/ComputeEngineIntegrationTest.java [hunk 3]
```diff
@@ -3,11 +3,6 @@
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.Test;
 
-import project.User;
-import project.ComputeEngine;
```
Comment (azip3) [inline test/project/ComputeEngineIntegrationTest.java:7]: Good. Better formatting would be having all imports at top together instead of space?

File: src/project/UserImp.java [hunk 4]
```diff
@@ -21,8 +21,8 @@ public ComputeResult compute(ComputeRequest request) {
 		DataStorageReadResult readResult =dataStorageAPI.read(inputConfiguration);		
 		Iterable<Integer> integer=readResult.getResults();
 		
-		if (integer!=null) {
-			return ComputeResult.SUCCESS;
+		if (integer==null) {
+			return ComputeResult.FAILURE;
```
Comment (azip3) [inline src/project/UserImp.java:25]: Good Job getting FAILURE to work instead of having the inverted version using SUCCESS.

File: src/project/ComputeResult.java [hunk 5]
```diff
@@ -8,6 +8,18 @@ public ComputeResultStatus getStatus() {
 			return ComputeResultStatus.SUCCESS;
 		}
 		
+		@Override
+		public String getFailureMethod() {
+			return "";
+		}
+	};
+	static ComputeResult FAILURE = new ComputeResult() {
+
```
Comment (azip3) [inline src/project/ComputeResult.java:17]: Good. NIT: formatting could be improved
